app/app/explore/components/SortHeader.tsx

```diff
@@ -0,0 +1,87 @@
+import { DownArrow } from "@shared/components/svg";
+import { useState, useRef, useEffect } from "react";
+
+interface Column {
+  key: string;
+  label: string;
+  width: string;
+  sortable?: boolean;
+  render: (item: any) => React.ReactNode;
+}
+
+export enum SortKey {
+  TVL = "tvl",
+  APR = "apr",
+}
+
+enum SortDirection {
+  ASC = "asc",
+  DESC = "desc",
+}
+
+interface SortConfig {
+  key: SortKey;
+  direction: SortDirection;
+}
+
+interface SortHeaderProps {
+  columns: Column[];
+  sortConfig: SortConfig;
+  onSort: (key: SortKey) => void;
+}
+
+const SortHeader: React.FC<SortHeaderProps> = ({
```

Comment:
IMO we should have a generic dropdown component that looks like this - https://www.figma.com/design/wIeUOaqwKmqzyhViQPKwce/Yieldi-Brand-%2B-Deck?node-id=2954-1423&t=GYddZWjFEikrl9nK-1

We'll have to use this dropdown in many other places, so it's good to have a single implementation of it.